packages/env/lib/build-docker-compose-config.js

```diff
@@ -113,6 +113,34 @@ module.exports = function buildDockerComposeConfig( config ) {
 	const developmentPorts = `\${WP_ENV_PORT:-${ config.env.development.port }}:80`;
 	const testsPorts = `\${WP_ENV_TESTS_PORT:-${ config.env.tests.port }}:80`;
 
+	// Set the WordPress, WP-CLI, PHPUnit PHP version if defined.
+	const phpVersion = config.env.development.phpVersion
+		? config.env.development.phpVersion.toString()
+		: '';
+	const wpImage = `wordpress${
+		! phpVersion || phpVersion.length === 0 ? '' : ':php' + phpVersion
+	}`;
+	const cliImage = `wordpress:cli${
+		! phpVersion || phpVersion.length === 0 ? '' : '-php' + phpVersion
+	}`;
+	// Defaults are to use the most recent version of PHPUnit that provides
+	// support for the specified version of PHP.
+	let phpunitVersion = '8';
```

Comment:
Nice! It's helpful to have this defined for different phpunit versions.

I wonder, should we just default it to "latest" unless there is a PHP version set?

Reply:
Yes, actually I am technically defaulting to latest here: https://github.com/WordPress/gutenberg/pull/25268/files#diff-fea7299342c14da6ab9a85568810c056R138-R142

I'm doing it this way since if there is a `phpVersion` set then the format of the image tag is very different from just using latest. Also, if I set `phpunitVersion` to `latest` by default then I will have to check more PHP versions to account for the range that uses PHPUnit 8. For the sake of clarity I decided to use a default of `latest` and change to include a single check for the 3 PHP versions that will use PHPUnit 8.

Reply:
You should see this change here: https://github.com/WordPress/gutenberg/pull/25268/files#diff-fea7299342c14da6ab9a85568810c056R131-R146